ld/nodemcu.ld

```diff
@@ -103,7 +103,11 @@ SECTIONS
     *(.init.literal)
     *(.init)
     
-    *sdk/esp_iot_sdk_*lib/lib*.a:*(.literal .text)
+    *libmain.a:*(.literal .text)
+    *libnet80211.a:*(.literal .text)
+    *libphy.a:*(.literal .text)
+    *libpp.a:*(.literal .text)
+    *libgcc.a:*(.literal .text)
```

Comment:
I don't think this list is complete. Have a look at the output of this:
```
nodemcu-firmware/sdk/esp_iot_sdk_v2.0.0/lib$ xtensa-lx106-elf-objdump -h *.a|grep -v '2\*\*0' | grep -v irom|grep -E 'format|.text|archive'
```
It seems the following libs also have some things in IRAM:
- ~~libat.a~~ (don't care, we don't use this)
- libcrypto.a
- libdriver.a (not sure if we're using any of this by now?)
- libespnow.a (not relevant yet, but might be worth listing here anyway)
- libmesh.a (as above)
- ~~liblwip_536.a~~ (crikey, espconn_buf.o has nearly 1.3k in IRAM! Lots of other bits too...)
- ~~libpwm.a~~ (we have our own pwm driver last I checked)
- libwpa.a
- ~~libwps.a~~ (intentionally excluded)